Makefile

```diff
@@ -66,19 +66,17 @@ INTEG_TEST_NDC_ROOT    := ./host/ndc
 PROTO_ROOT := proto
 PROTO_FILES = $(shell find ./$(PROTO_ROOT)/internal -name "*.proto")
 PROTO_DIRS = $(sort $(dir $(PROTO_FILES)))
-PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(GOPATH)/src/github.com/temporalio/gogo-protobuf/protobuf
+PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell cd build && go list -m -f '{{.Dir}}' github.com/temporalio/gogo-protobuf)/protobuf
 PROTO_OUT := api
 
-
 ALL_SRC         := $(shell find . -name "*.go")
 ALL_SRC         += go.mod
-# Replace below with build tags and `go test ./...` for targets
+ALL_SCRIPTS     := $(shell find . -name "*.sh")
+# TODO (jeremy): Replace below with build tags and `go test ./...` for targets
```

Comment:
I think he's probably not going to do this anymore...

Reply:
```
$ go list -f '{{.Dir}}' github.com/temporalio/gogo-protobuf/protobuf
no required module provides package github.com/temporalio/gogo-protobuf/protobuf; to add it:
        go get github.com/temporalio/gogo-protobuf/protobuf
$ go get github.com/temporalio/gogo-protobuf/protobuf
go: module github.com/temporalio/gogo-protobuf@upgrade found (v1.22.1), but does not contain package github.com/temporalio/gogo-protobuf/protobuf
```
:-(

Reply:
I hope I will.